db/version_set.cc

```diff
@@ -1310,7 +1310,8 @@ void VersionStorageInfo::ComputeCompactionScore(
       } else {
         score = static_cast<double>(num_sorted_runs) /
                 mutable_cf_options.level0_file_num_compaction_trigger;
-        if (compaction_style_ == kCompactionStyleLevel && num_levels() > 1) {
+        if (score >= 1.0 && compaction_style_ == kCompactionStyleLevel &&
+            num_levels() > 1) {
```

Comment:
In line 1320, can we use mutable_cf_options.max_bytes_for_level_base instead of MaxBytesForLevel(base_level())?